Bottom: 4cca00de9771567e243f383e4ef1aa1bb597a8f4
Top:    1d8464257d48ab2bc18e61d0f8312b34b8731fa4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:20:15 +0000

Refresh of continue-tinkering-with-3d

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 2b42e3a..720543f 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -297,7 +297,7 @@ get_contour_coord_n_in_mm (PLINE *contour, int n, double *x, double *y)
 
 
 void
-object3d_export_to_step (object3d *object, char *filename)
+object3d_export_to_step (object3d *object, const char *filename)
 {
   FILE *f;
   time_t currenttime;
diff --git a/src/hid/common/object3d.h b/src/hid/common/object3d.h
index ecdc721..f396295 100644
--- a/src/hid/common/object3d.h
+++ b/src/hid/common/object3d.h
@@ -17,5 +17,5 @@ void object3d_add_vertex (object3d *object, vertex3d *vertex);
 void object3d_add_face (object3d *object, face3d *face);
 object3d *object3d_create_test_cube (void);
 GList *object3d_from_board_outline (void);
-void object3d_export_to_step (object3d *object, char *filename);
+void object3d_export_to_step (object3d *object, const char *filename);
 void object3d_test_board_outline (void);
